src/line-chart/index.js

```diff
@@ -0,0 +1,3 @@
+import LineChart from "./line-chart";
+
+export default LineChart;
```

Comment:
I made this a default export, so that there is no breaking changes for current library users.

Reply:
Apologies, should have caught that before submitting. Thanks for sorting it out!